logs/logs.go

```diff
@@ -0,0 +1,65 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
```

Comment:
This doesn't make any sense.  This is supposed to return a key, not the token itself.  I'm surprised this works